package.json

```diff
@@ -8,19 +8,17 @@
     "@types/react": "^17.0.43",
     "@types/react-dom": "^17.0.14",
     "@types/react-redux": "^7.1.23",
-    "@types/reselect": "^2.2.0",
     "prettier": "^2.6.1",
     "react-scripts": "^4.0.3",
     "react-test-renderer": "^16.13.1"
   },
   "dependencies": {
+    "@reduxjs/toolkit": "^1.8.1",
     "classnames": "^2.2.6",
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-redux": "^7.2.0",
-    "redux": "^4.0.5",
-    "reselect": "^4.0.0",
```

Comment:
`redux` and `reselect` are not needed as direct dependencies because they are exported from `@redux/toolkit`